modules/markup/common/footnote.go

```diff
@@ -232,7 +232,7 @@ func (b *footnoteBlockParser) Open(parent ast.Node, reader text.Reader, pc parse
 	return item, parser.HasChildren
 }
 
-func (b *footnoteBlockParser) Continue(node ast.Node, reader text.Reader, pc parser.Context) parser.State {
+func (b *footnoteBlockParser) Continue(_ ast.Node, reader text.Reader, _ parser.Context) parser.State {
```

Comment:
Here `Continue(node ast.Node` and above `Open(parent ast.Node`, after the refactoring, it seems more difficult to understand.

Is it possible to use `_node ast.Node` and `_parent ast.Node` to pass the lint?

Reply:
> Is it possible to use _node ast.Node and _parent ast.Node to pass the lint?

```
modules/markup/common/footnote.go:286:32: parameter '_node' seems to be unused, consider removing or renaming it as _
```

Unfortunately not, but we can make use of here by removing `_` such that only the type is displayed. Or we could use `//revive:disable.....`
But that is just more hacks IMO

Reply:
Well .... maybe we can suggest the lint authors to accept such styles .... let lint users decide whether they accept `_var` variables.

Reply:
Agree, would prefer `_node` over `_` because it preserves the meaning while marking the parameter unused. We have this exact setup also for JS:

https://github.com/go-gitea/gitea/blob/49933c7fb09cc98ae0e57a483d68011ac67fd492/.eslintrc#L305

Is it possible to configure the go linter like that as well?

Reply:
Yeah that would makes that it could be configured within the settings, I've opened a Issue for it https://github.com/mgechev/revive/issues/613

Reply:
> Well .... maybe we can suggest the lint authors to accept such styles .... let lint users decide whether they accept `_var` variables.

Not much luck on that one.

```suggestion
func (b *footnoteBlockParser) Continue(ast.Node, reader text.Reader, parser.Context) parser.State {
```
I think such thing can be used to have the readability.
```

Reply:
```go
func (b *footnoteBlockParser) Continue(ast.Node, reader text.Reader, parser.Context) parser.State 
```

I'm not sure that you can mix named and unnamed parameters

Reply:
It's still better to keep the names.

`Open(parent ast.Node, ...`
`Continue(node ast.Node, ...`

They have different meanings.

Reply:
> They have different meanings.

These parameters mean nothing to those methods (because they are unused)
What is the interest of documenting (putting a name to) something that is not used?

As user of the method it seems, to me, more clear a `_` than an actual name. 
`_` means "_do not take care of this parameter, it has no influence in the method behavior_", then as user of the method I know (instantly) that I can pass anything as argument. Contrary, if the parameter has a name, I will need to take care of providing an argument that fits the expectations (e.g. a valid `context` or an actual _parent_ `ast.Node`)... for nothing because the parameter is not used.